helm/charts/infra/values.yaml

```diff
@@ -5,17 +5,15 @@ nameOverride: ""
 fullnameOverride: ""
 
 global:
```

Comment:
The structure of this object shouldn't change since it may break existing deployments. The UI image.repository doesn't need to use any global values. See #2602 which adds a postgres component to the change that's configured entirely separately from the main infra apps

Reply:
Gotcha. Should we still keep the global object around if some services won't use it?

Reply:
There is. The main image is still shared so changing it in all places without a global would be painful. It's completely fine to have some deployments not use the global image